package.json

```diff
@@ -17,6 +18,7 @@
   "dependencies": {
     "@vue/babel-preset-app": "^4.5.10",
     "compression-webpack-plugin": "^6.0.4",
+    "husky": "^7.0.1",
```

Comment:
@vickywane shouldn't husky be a devDependency?

Reply:
It can also be used in that form. 

I have re-installed it as a devDependency.

Reply:
ok